frontend/src/hooks/useGetBoardInfo.ts

```diff
@@ -83,31 +96,46 @@ export const useGetBoardInfoEffect = (): useGetBoardInfoInterface => {
           .getBoardInfo(boardInfoRequest)
           .then((res) => {
             if (!res.data) {
-              setErrorMessage({
-                title: BOARD_CONFIG_INFO_TITLE.NO_CONTENT,
-                message: BOARD_CONFIG_INFO_ERROR.NOT_CONTENT,
-                code: HttpStatusCode.NoContent,
-              });
+              errorCount++;
+              localBoardInfoFailedStatus = METRICS_DATA_FAIL_STATUS.PARTIAL_FAILED_NO_CARDS;
+              setBoardInfoFailedStatus(METRICS_DATA_FAIL_STATUS.PARTIAL_FAILED_NO_CARDS);
             }
             return res;
           })
           .catch((err) => {
             const { code } = err;
```

Comment:
the `err` is not 100% sure to be an object, so destruct on an `unknown` type potentially breaks your logic, which may not go to line-103 `return res`. Furtherly, not get into `Promise.all`.